### PR TITLE
Lazy repeat

### PR DESCRIPTION
**Pull request by argelius**
@anatoo さん

iOSだとまだ問題があります。iOSでスクロールしている間、全然レンダリングしてくれないです。
強制的にレンダリングをさせてみましたが、どんな方法でもスクロールが終わってからレンダリングします。

### CODE REVIEW

**Comment by argelius** [comment]
@anatoo 

I changed so it will re-render the content every time it's updated so it can be used with `ons-pull-hook`. 
In the Chrome CPU profiler i get 80% idle time so it seems fast enough.

**Comment by anatoo** [comment]
Great! 
I'm going to review your code and write Japanese doc comments.

**Comment by argelius** [comment]
どうもありがとうございます

File: framework/views/lazyRepeat.js [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+        $document[0].addEventListener('scroll', this._bindedOnScroll, true);
+      },
+
+      _removeEventListeners: function() {
+        $document[0].removeEventListener('scroll', this._bindedOnScroll, true);
+      },
+      
+      _destroy: function() {
+        this._removeEventListeners(); 
+        this._element = this._scope = this._attrs = null;
```
Comment (anatoo) [inline framework/views/lazyRepeat.js]: Request: It is seem that "this._parentElement" and "this._renderedElements" is not released. Could you confirm that it is no DOM leaks?

**Comment by Hirbod** [comment]
Seems this PR is just usable for list-items, not for normal content. That's a bit sad.
Maybe you could a small look on this:
https://github.com/lorenzofox3/lrInfiniteScroll

I've managed to get this work very well inside of OnsenUI

**Comment by argelius** [comment]
It should work fine for any element as long as you know their height.

**Pull request merged**